tests/h/traversal/roots_test.py

```diff
@@ -76,24 +79,85 @@ def links_service(self, pyramid_config):
         return service
 
 
-class TestAuthClientResourceFactory(object):
-    def test_get_item_returns_an_authclient(self, pyramid_request):
-        authclient = AuthClient(name='test', authority='example.com')
-        pyramid_request.db.add(authclient)
-        pyramid_request.db.flush()
+class TestAuthClientRoot(object):
+    def test_getitem_returns_the_right_AuthClient(self, db_session, pyramid_request):
+        # Add a couple of noise AuthClients to the DB. It should not return these.
```

Comment:
Big thumbs up on adding explanatory note comments in tests. It helps the next dev a ton.